packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -4263,6 +4342,10 @@ export class ContainerRuntime
 				const idAllocationBatchMessage: LocalBatchMessage = {
 					runtimeOp: idAllocationMessage,
 					referenceSequenceNumber: this.deltaManager.lastSequenceNumber,
+					//* Need tests for this - is it an ok strategy?
+					// Note: For now, we will never stage ID Allocation messages.
+					// They won't contain personal info and no harm in extra allocations in case of discarding the staged changes
+					staged: false,
```

Comment:
@Abe27342 this is the ID Compressor thing. What happens if "unused" ID Allocation ops are submitted?

Reply:
Nothing bad per se. The reconnect strategy id compressor uses is basically this. That said, since it's a little unorthodox, it might be worth chatting about. I'll scrutinize some of the relevant code paths on this PR.